geopackage_validator/constants.py

```diff
@@ -0,0 +1,24 @@
+ALLOWED_LIST = [
```

Comment:
Misschien even hernoemen naar ALLOWED_PROJECTIONS_LIST omdat ie nu in de algemene constants staat

Reply:
Of iets vergelijkbaars